source/Engine.cpp

```diff
@@ -34,6 +34,7 @@ void Engine::Update()
 		double frameTime = currentTime - myPreviousTime;
 		myPreviousTime = currentTime;
 
+		myJobSystem.Update(frameTime);
```

Comment:
We can use forward declarations and make this a pointer, so that the Engine class doesn't become too big to include.